src/api/index.js

```diff
@@ -34,11 +36,17 @@ const getLedger = require('./ledger/ledger');
 
 function RippleAPI(options: {}) {
   common.validate.remoteOptions(options);
+  EventEmitter.call(this);
   const _options = _.assign({}, options, {automatic_resubmission: false});
   this.remote = new common.core.Remote(_options);
+  this.remote.on('ledger_closed', message => {
+    this.emit('ledgerClosed', message);
```

Comment:
Maybe RippleAPI should format the message to be consistent with camelcasing convention https://github.com/ripple/ripple-lib/blob/develop/test/fixtures/api/rippled/ledger-close.json